api/src/nv_ingest_api/internal/extract/pdf/engines/pdfium.py

```diff
@@ -551,8 +536,6 @@ def pdfium_extractor(
             if extract_tables or extract_charts or extract_infographics:
                 image, padding_offsets = pdfium_pages_to_numpy(
                     [page],
-                    scale_tuple=(YOLOX_PAGE_IMAGE_PREPROC_WIDTH, YOLOX_PAGE_IMAGE_PREPROC_HEIGHT),
-                    padding_tuple=(YOLOX_PAGE_IMAGE_PREPROC_WIDTH, YOLOX_PAGE_IMAGE_PREPROC_HEIGHT),
```

Comment:
We might need to keep this because some pages can be very large which we would want to downscale before sending it to NIM.